qiskit/schemas/qobj_schema.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "http://json-schema.org/draft-04/schema#",
     "id": "http://www.qiskit.org/schemas/qobj_schema.json",
     "description": "OpenQuantum quantum object data structure for running experiments",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
I was going to ask if you could update the outbound payload version (assuming we're sending `discriminator` and `kernel` in the qobj) so when there is proper schema version support the server know what we're sending. But it looks like that's already set to `1.1.0`:

https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/qobj/pulse_qobj.py#L515
https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/qobj/qasm_qobj.py#L408

Reply:
So do we need to bump again to `1.2.0`?